libpod/container_log.go

```diff
@@ -53,6 +63,18 @@ func (r *Runtime) Log(containers []*Container, options *LogOptions, logChannel c
 
 // ReadLog reads a containers log based on the input options and returns loglines over a channel
 func (c *Container) ReadLog(options *LogOptions, logChannel chan *LogLine) error {
+	// TODO Skip sending logs until journald logs can be read
+	// TODO make this not a magic string
+	if c.LogDriver() == JournaldLogging {
```

Comment:
@baude We support follow with journald events, right? How bad would this be to add?

Reply:
@mheon I can't find the code that follows for journald events. What I do know is the sdjournal.Follow function spits the output to a writer, so we'd have to implement a buffered reader that waited for a newline, then put that line for processing as a LogLine. Not a horrible amount of work, but definitely clunky. If the feature is a blocker I can implement it.

Reply:
@mheon just kidding it wasn't that hard to do, I just misunderstood the api